### DISCUSSION TITLE
How would I go about settingState in the onChange function, I have a multi-select and want to store the values in state

### DISCUSSION THREAD

**Discussion by RyanSpeciale** [JedWatson/react-select#5336] [Q&A]
Hi I am just wondering how I would save the array to state in Typescript in a multi-select

**Comment by manjushsh** [comment]
Hi @RyanSpeciale, When you do setState on multi select, values are stored as array by default.
Please [refer this](https://react-select.com/typescript#onchange) 
You can check [this example](https://codesandbox.io/s/react-select-examples-6picqk?file=/src/another-select.tsx)
